src/api/converter.ts

```diff
@@ -1,6 +1,34 @@
-import { DocumentSummary } from './types';
+import { ProjectSummary, DocumentSummary } from './types';
 
-import { DocumentSummary as PbDocumentSummary } from './resources_pb';
+import {
+  Project as PbProjectSummary,
```

Comment:
`ProjectSummary` seems to be good to just use `Project`.
We use `DocumentSummary` to distinguish it from the actual document.

```suggestion
  Project as PbProject,
```